cosmos/haqq_11235.json

```diff
@@ -28,6 +28,12 @@
       "coinDecimals": 18,
       "coinGeckoId": "islamic-coin",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/haqq_11235/islm.png"
+    },
+    {
+      "coinDenom": "DEEN",
+      "coinMinimalDenom": "erc20/0x4FEBDDe47Ab9a76200e57eFcC80b212a07b3e6cE",
```

Comment:
Hi. Is this token denom used on Cosmos node or EVM node?

Reply:
@editaahn This is a cosmos token - our blockchain - HAQQ has an EVM module, and this is a token created on the EVM layer and registered as a Cosmos token. 

Here it is on Osmosis:
https://app.osmosis.zone/pool/2052

https://github.com/osmosis-labs/assetlists/blob/ee60195ab6ca6a0904a1a092ba4fc44408da2969/osmosis-1/osmosis.zone_assets.json#L4936-L4948

Here it is on Keplr: 
<img width="381" alt="Screenshot 2024-08-29 at 06 02 50" src="https://github.com/user-attachments/assets/0ccf735f-892e-4aa0-9043-06fd1c651036">